src/components/TimeRemaining.js

```diff
@@ -21,10 +28,40 @@ export default function TimeRemaining({ activities }) {
     }, 0)
     return 24 - sum
   }
+
+  function hourOverflow(activities) {
+    const sum = activities.reduce((sum, activityItem) => {
+      return sum + activityItem.hours
+    }, 0)
+    return sum - 24 + 1
+  }
```

Comment:
This is very similar to `remainingHours(` maybe extract the sum calculation to its own function, or reuse `remainingHours(` in `hoursOverflow(`